src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,11 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
```

Comment:
Don't begin class or method implementation with empty line. Remove the empty line at the start of the class definition.